programmers/난이도별/level01.최대공약수와_최소공배수/khw970421.js

```diff
@@ -11,11 +11,14 @@ function solution(n, m) {
 
     return [a].concat([m*n/a])    // a%b가 0이되었을때의 나누는수 b는 a=b로 인해 a이므로 최대공약수는 a 최소공배수는 m*n/a이다.
```

Comment:
```suggestion
    return [a, m*n/a]
```
concat 을 쓰지 않아도 될 것 같습니다

Reply:
10번 줄 에서
    while(r!=0)
비교대신 
    `while(r!==0)` 타입까지 비교를 해 주는 것이 권장사항 입니다

Reply:
충분히 함수를 안쓰고도 [ , ]를 통해도 가능하군요 , 타입비교까지 확인했습니다.